vendor/bou.ke/monkey/LICENSE.md

```diff
@@ -0,0 +1,5 @@
+Copyright Bouke van der Bijl
+
+I do not give anyone permissions to use this tool for any purpose. Don't use it.
+
+I’m not interested in changing this license. Please don’t ask. 
```

Comment:
That’s not good.

Reply:
This is the limitation of `go`: there are no other tools for monkey patching (as I know) and it is forbidden by language itself. I don't know why you are using `go`.

Reply:
I will try to write parameters version of function to test it without monkey patching.